ldk/javascript/src/promisify.ts

```diff
@@ -30,6 +30,54 @@ export function promisifyMappedBothWithParams<TParamIn, TParamOut, TInternalOut,
   });
 }
 
+export function promisifyMappedWithTwoParams<TParam1, TParam2, TInternalOut, TExternalOut>(
+  param1: TParam1,
+  param2: TParam2,
+  map: Mapper<TInternalOut, TExternalOut>,
+  arg: OliveHelps.ReadableWithTwoParams<TParam1, TParam2, TInternalOut>,
+): Promise<TExternalOut> {
+  return new Promise((resolve, reject) => {
```

Comment:
I really need to spend some time looking into these types and figure out if there's an easier way to express all these functions.